go/vt/topo/topoproto/tablet.go

```diff
@@ -63,6 +63,16 @@ func TabletAliasEqual(left, right *topodatapb.TabletAlias) bool {
 	return proto.Equal(left, right)
 }
 
+// TabletInList returns true if the tablet is in the list of tablets given
```

Comment:
Suggestion: rename to `IsTabletInList` to match `func IsTypeInList(tabletType topodatapb.TabletType, types []topodatapb.TabletType) bool {`

Reply:
Fixed